src/auth/oauth/mod.rs

```diff
@@ -413,9 +415,9 @@ async fn init_oauth_code_flow(
     let redirect_uri = append_params_to_uri(&redirect_uris.validated.0, &redirect_params);
 
     // IETF RFC 6749 Section 4.1.2 (https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2)
-    Ok(HttpResponse::Found()
-        .append_header((LOCATION, redirect_uri))
-        .finish())
+    Ok(HttpResponse::Ok()
```

Comment:
See above
the spec seems to say should be a 302- if there is an issue with it for our setup I would recommend leaving a comment here saying this deviates from the spec (we shouldnt have the spec as a comment right above a place it deviates from it)